app/views/conference/_all_events.html.erb

```diff
@@ -1,19 +1,73 @@
 <h1 class="text-center">Program for <%= @conference.title %></h1>
-<% @conference.program.events.confirmed.each do |event| %>
-    <div>
-        <h3>
-          <%= link_to event.title, conference_program_proposal_path(@conference.short_title, event.id) %>
-          <br>
-          <small>
-            <%= event.subtitle %>
-          </small>
-        </h3>
-        <h4>
-          presented by <%= event.speaker_names %>
-        </h4>
-        <p>
-          <%= truncate(event.abstract, :length => 400) -%>
-          <%= link_to 'more', conference_program_proposal_path(@conference.short_title, @conference.program.id, event.id) if event.abstract.length > 400 %>
-        </p>
+<div class="dropdown program-dropdown">
```

Comment:
I think this is not needed. Just do simple anchors for the days and events like this:

![ezgif com-video-to-gif 3](https://cloud.githubusercontent.com/assets/514785/16084075/a737dfd2-3317-11e6-94ac-ad209917470d.gif)